tests/test_testPipeline.py

```diff
@@ -141,6 +153,27 @@ def setUp(self):
         super().setUp()
         self.butler = butlerTests.makeTestCollection(self.repo, uniqueId=self.id())
 
+    def testMockLoadDiaCatalogsTask(self):
+        config = MockLoadDiaCatalogsTask.ConfigClass()
+        config.apdb_config_url = "testing_only"
+        task = MockLoadDiaCatalogsTask(config=config)
+        pipelineTests.assertValidInitOutput(task)
+        points = [UnitVector3d.Z(), UnitVector3d.X(), UnitVector3d.Y()]
+        region = lsst.sphgeom.ConvexPolygon(points)
```

Comment:
If you're making a dummy input, `sphgeom.Circle` has [a default constructor](http://doxygen.lsst.codes/stack/doxygen/x_mainDoxyDoc/classlsst_1_1sphgeom_1_1_circle.html#acc3ef9b79e6d97e854f9ee1eed81c42c) so you don't have to import `UnitVector3d`. You get an empty region either way.